ghcide/src/Development/IDE/Core/Tracing.hs

```diff
@@ -1,11 +1,13 @@
+{-# LANGUAGE CPP #-}
 {-# LANGUAGE DataKinds #-}
+#include "ghc-api-version.h"
```

Comment:
Should we drop this now we don't have ghc-lib support?

Reply:
Or do we still have ghc-lib support? I'm confused...

Reply:
we still do, but it's not being tested so it might have bit rotten